docs/v1/P4Runtime-Spec.adoc

```diff
@@ -1757,31 +1573,35 @@ Both `Meter` and `DirectMeter` messages share the following fields:
   this meter extern instance. The `MeterSpec` message is used to describe the
   meter unit and the meter type. The meter unit can be any of the
   `MeterSpec.Unit` enum values:
-    * `UNSPECIFIED`: reserved value.
-    * `BYTES`, which signifies that this meter can be configured with rates
+    ** `UNSPECIFIED`: reserved value.
+    ** `BYTES`, which signifies that this meter can be configured with rates
       expressed in bytes/second.
-    * `PACKETS`, for rates expressed in packets/second.
-
-  The meter type can be any of the `MeterSpec.Type` enum values:
+    ** `PACKETS`, for rates expressed in packets/second.
+    +
```

Comment:
I am not certain, but I believe that `+` continuation lines like this need to have no whitespace.